wpimath/src/main/native/include/frc/trajectory/Trajectory.h

```diff
@@ -66,7 +109,14 @@ class WPILIB_DLLEXPORT Trajectory {
    *
    * @throws std::invalid_argument if the vector of states is empty.
    */
-  explicit Trajectory(const std::vector<State>& states);
+  explicit Trajectory(const std::vector<State>& states) : m_states(states) {
```

Comment:
These were moved to the header but are not constexpr?

Reply:
They originally were, but we need GCC 12 for constexpr std::vector. Bumping our RPi OS version would fix that.